contrib/ci/fwupd_setup_helpers.py

```
@@ -72,13 +72,21 @@ def detect_profile():
+    env = os.environ.copy()
+    if use_pipx and shutil.which("pipx"):
+        env["PIPX_HOME"] = "/opt/pipx"
+        env["PIPX_BIN_DIR"] = "/usr/bin"
+        cmd = ["pipx", "install", "--force", name]
+    else:
+        env["PIP_BREAK_SYSTEM_PACKAGES"] = "1"
+        cmd = ["python3", "-m", "pip", "install", "--upgrade", name]
     if debug:
         print(cmd)
-    rc = subprocess.call(cmd)
+    rc = subprocess.call(cmd, env=env)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)